lib/Driver/ToolChain.cpp

```diff
@@ -280,14 +280,16 @@ mergeBatchInputs(ArrayRef<const Job *> jobs,
   return false;
 }
 
-/// Debugging only: return whether the set of \p jobs is an ordered subsequence
-/// of the sequence of top-level input files in the \c Compilation \p C.
-static bool
-jobsAreSubsequenceOfCompilationInputs(ArrayRef<const Job *> jobs,
-                                      Compilation &C) {
-  llvm::SmallVector<const Job *, 16> sortedJobs;
+/// Due to SR-631 / rdar://20535574, the success or failure of a Swift compilation
+/// is currently sensitive to e order in which files are processed, in terms of
```

Comment:
Typo: "e order". I also think citing a specific SR / Radar isn't so great here—that's the one we know about, but we'd want to do experiments to make sure there wasn't *any* order-dependency left in the non-incremental parts of the work.

Sorry for all this nitpicking!

Reply:
@jrose-apple S'ok, as I've said before, you're picky but I can't remember a nit you've picked that didn't need picking; better than my sloppy reviewing style! And being subject to it is a good exercise in cultivating equanimity anyway: the PR will land when it's ready, and not before :)

Meanwhile: pushed a fix.